sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -340,6 +343,28 @@ for invoice in result:
         print("{}: {} has confidence {}".format(name, field.value, field.confidence))
 ```
 
+### Recognize Id documents
+Recognize data from id documents using a prebuilt model. Id document fields recognized by the service can be found [here][service_recognize_id_documents].
+
+```python
+from azure.ai.formrecognizer import FormRecognizerClient
+from azure.core.credentials import AzureKeyCredential
+
+endpoint = "https://<region>.api.cognitive.microsoft.com/"
+credential = AzureKeyCredential("<api_key>")
+
+form_recognizer_client = FormRecognizerClient(endpoint, credential)
+
+with open("<path to your id document>", "rb") as fd:
+    id_document = fd.read()
+
+poller = form_recognizer_client.begin_recognize_id_documents(id_document)
+result = poller.result()
+
+for id_document in result:
+    for name, field in id_document.fields.items():
+        print("{}: {} has confidence {}".format(name, field.value, field.confidence))
+```
```

Comment:
we've talked about refactoring the examples section once we had several prebuilts since these samples all pretty much look the same and make the readme pretty lengthy. not blocking on this PR, but maybe open an issue so we can discuss how we want to do it across langs?

Reply:
Agreed! I didnt want to skip id docs just now since all the other models still have their examples. Here's the issue: #17666